### DISCUSSION TITLE
Looking for research opportunities

### DISCUSSION THREAD

**Discussion by nyrareddy** [fastmachinelearning/hls4ml#920] [General]
Are there any research projects at hls4ml open for students? I tried looking for a post related to [this](https://github.com/fastmachinelearning/hls4ml/discussions/857#discussioncomment-6840960) but I'm not sure I can find it.

**Comment by pierinim** [comment]
Hello
There is a five-year project starting at CERN in 2024 that foresees funds for hls4ml developers as well as  to develop applications based on hls4ml  for the atlas and cms experiments at the cern large hadron collider.

There will be openings at various level for internships at CERN (typically one year for undergraduate students, three years for graduate students) as well as post-doctoral positions. Details are being discussed. We will post something here and on Linkedin when we will know the actual details.

My suggestion is to start preparing an application file for a technical (bachelor or master) or doctoral (PhD) students, explicitly mentioning FPGA, ML, AI and hls4ml in your 
application, in order to facilitate our search for candidates. Refer to previous  openings of similar kind

https://careers.smartrecruiters.com/CERN/students

Good luck

**Reply by dmasloub** [reply]
Hello
I’m interested in a 13-week summer internship with the hls4ml project at CERN. I used hls4ml extensively for my bachelor’s thesis and would love to contribute during my master’s studies. A one-year internship sadly isn’t feasible for me.
Could you let me know if shorter internships are also available?

**Reply by nyrareddy** [reply]
Hello @pierinim @vloncar, sorry for replying to this old thread. But are there any current openings/research projects for graduate students?

**Reply by vloncar** [reply]
You should look at [CERN's careers page](https://careers.cern/), all jobs will eventually be listed there. You can browse based on your experience level (student, graduate, professional etc). There are multiple hiring rounds per year for students. Based on openings in our projects we select students from that pool. We value experience with FPGAs and hls4ml so if you have some make sure you mention it in your application to stand out more.
